CakePHP/Sniffs/Commenting/FunctionCommentSniff.php

```diff
@@ -269,10 +268,9 @@ protected function processThrows(File $phpcsFile, $stackPtr, $commentStart)
                     $error = '@throws tag comment must end with a full stop';
                     $phpcsFile->addWarning($error, ($tag + 2), 'ThrowsNoFullStop');
                 }
-            }//end if
-        }//end foreach
-    }//end processThrows()
```

Comment:
Why have you removed all these really useful comments? :trollface:

Reply:
I almost thought about adding also `;//end of statement` into the other lines for consistency.

Reply:
I'm pretty sure these came for the ride when we pulled in from squizlabs.